cmd/pomerium-cli/kubernetes.go

```diff
@@ -14,7 +14,7 @@ import (
 
 	"github.com/skratchdot/open-golang/open"
 	"github.com/spf13/cobra"
-	"github.com/square/go-jose/jwt"
+	jose "github.com/square/go-jose/v3"
```

Comment:
Confusingly we use "gopkg.in/square/go-jose.v2/jwt" elsewhere already which is the package name recommended in their docs (though I'm not really sure of the difference). 

Thoughts on using v2 vs v3 which is marked as [unstable for the time being](https://github.com/square/go-jose#versions)?

Reply:
The jwt package didn't work for me. (from what I could tell we have to use the Claims call which requires the secret key, which we don't have) They switched from gopkg because its no longer needed with go modules. I can try to use the v2 package if we'd rather use that.

Reply:
> The jwt package didn't work for me.

Yeah, sorry, I mean the go-jose package in general (including the jwt sub-package). 

> They switched from gopkg because its no longer needed with go modules

I'm fine either way, but would like to be consistent. Should we change all references then so don't have multiple versions of the same package (both `go-jose` and `go-jose/jwt`? 

> I can try to use the v2 package if we'd rather use that.

I think until v3 is marked stable, I think it makes sense to do that.